azdev/utilities/command.py

```diff
@@ -31,9 +32,13 @@ def call(command, **kwargs):
     :param kwargs: Any kwargs supported by subprocess.Popen
     :returns: (int) process exit code.
     """
-    return subprocess.call(
-        command,
-        shell=True,
+    from azdev.utilities import IS_WINDOWS
+    cmd_args = shlex.split(command)
+    if cmd_args[0] == "az" and IS_WINDOWS:
+        cmd_args[0] = "az.bat"
+    return subprocess.run(
+        cmd_args,
+        check=False,
```

Comment:
`check=False` is by default.

Reply:
yes, but linter will raise (subprocess-run-check) error if not set explicitly

Reply:
Can we add a comment stating this?

Reply:
added